icons.json

```diff
@@ -0,0 +1 @@
.414v6.048H2.62V3.414zm11.818 11.389c-1.021 0-1.849-.79-1.849-1.762 0-.971.828-1.76 1.849-1.76 1.02 0 1.847.789 1.847 1.76 0 .973-.828 1.762-1.847 1.762z","legacy":true},"amenitySpa":{"viewBox":"0 0 20 16","path":"M4.08 3.746c3.46.099 6.394 3.548 6.02 6.924-3.045.365-4.442-1.232-6.02-6.924zM10.922 10.686c-.249-1.636-.62-3.15.748-4.479 1.282-1.247 2.526-2.448 4.48-2.516-.63 4.802-1.93 6.559-5.228 6.995z M1.695 9.633c-1.052 1.524-.298 2.458 1.056 3.166 2.931 1.532 5.944 2 9.152 1.59 2.075-.266 4.15-.891 6.092-1.678 1.316-.562 1.581-1.42.904-2.845 1.013.697 1.184 1.84.478 2.69-.682.82-1.453 1.156-2.425 1.544-4.702 1.877-9.44 1.877-14.09-.241-.664-.302-1.287-.813-1.773-1.363-.951-1.08-.696-2.086.606-2.864zM10.138.394c1.201.808 1.553 2.086 1.873 3.364.399 1.594-1.276 2.185-1.633 3.456-2.947-2.781-2.582-4.2-.24-6.82z M3.845 10.294c2.063 2.517 10.52 2.479 12.642.098-.166 1.646-2.113 2.554-5.79 2.698-3.768.151-6.494-.899-6.852-2.796z","legacy":true},"amenityWifi":{"viewBox":"0 0 20 14","path":"M18.756 5.353a12.476 12.476 0 0 0-1.826-1.506 12.57 12.57 0 0 0-2.107-1.142 12.306 12.306 0 0 0-4.827-.977 12.413 12.413 0 0 0-4.82.965c-.74.311-1.442.69-2.103 1.138a12.78 12.78 0 0 0-1.838 1.504L0 4.095a14.618 14.618 0 0 1 2.096-1.702 14.093 14.093 0 0 1 2.399-1.289 14.267 14.267 0 0 1 2.65-.817A13.92 13.92 0 0 1 9.998 0c.976 0 1.928.094 2.854.286.927.19 1.813.464 2.659.818A14.06 14.06 0 0 1 20 4.114l-1.244 1.239zm-2.633 2.613A8.845 8.845 0 0 0 13.37 6.12a8.564 8.564 0 0 0-1.629-.503 8.777 8.777 0 0 0-1.745-.172c-.597 0-1.179.057-1.743.172a9.02 9.02 0 0 0-1.624.494 8.225 8.225 0 0 0-1.465.784A8.72 8.72 0 0 0 3.887 7.94L2.668 6.734a10.305 10.305 0 0 1 3.29-2.204 10.366 10.366 0 0 1 4.04-.813 10.309 10.309 0 0 1 5.808 1.775 10.64 10.64 0 0 1 1.536 1.26l-1.22 1.214zm-2.582 2.57a5.133 5.133 0 0 0-1.598-1.087 4.805 4.805 0 0 0-1.946-.396c-.694 0-1.34.127-1.94.383a5.1 5.1 0 0 0-1.589 1.067L5.301 9.34a6.739 6.739 0 0 1 2.114-1.403c.8-.34 1.66-.51 2.582-.51a6.602 6.602 0 0 1 4.712 1.948l-1.168 1.163zM10.012 14l-2.215-2.19a3.33 3.33 0 0 1 .998-.654c.375-.16.775-.24 1.202-.24.432 0 .835.083 1.213.248.378.167.706.392.984.68L10.012 14z","legacy":true},"arrowDirectional":{"viewBox":"0 0 20 9","path":"M19.79 4.147c-.052-.053-.079-.105-.13-.158L15.752.139a.522.522 0 0 0-.707 0c-.184.21-.184.527 0 .712l3.171 3.138H.498A.517.517 0 0 0 0 4.49c0 .264.236.501.498.501h17.72L15.045 8.13a.503.503 0 0 0 0 .713.516.516 0 0 0 .367.158.497.497 0 0 0 .34-.132l3.906-3.85.158-.158c.104-.106.183-.212.183-.343a.653.653 0 0 0-.21-.37z","legacy":true},"arrowExpander":{"viewBox":"0 0 24 24","path":"M19.4840214,7.20598808 L19.4832811,5.58543204 L15.2680867,9.85739783 C14.8742195,10.2561596 14.2364062,10.2565348 13.8429091,9.85739783 C13.4486716,9.458636 13.4486716,8.81153898 13.8421687,8.41277715 L18.0581034,4.14081137 L16.4596833,4.14081137 C15.9033088,4.1411865 15.451694,3.68352947 15.451694,3.11933589 C15.4524344,2.55551743 15.9033088,2.09823553 16.4596833,2.0978604 L20.4920107,2.0978604 C21.0483
```

Comment:
This is probably left over from the master branch. This file isn't needed in the `next` branch's setup so this should be removed